### PR TITLE
CASMINST-6208 bump cray-iuf, cray-nls, and iuf-cli versions

### PR DESCRIPTION
**Pull request by leliasen-hpe**
## Summary and Scope

Bump cray-iuf, cray-nls, and iuf-cli versions.
This impliments all changes made for CASMINST-6208.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by cdelatte-hpe — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+- Update iuf-cli from 1.5.2 to 1.5.3 and cray-nlsand cray-iuf from 3.1.1 to 3.1.2 (CASMINST-6208)
```
Comment (bo-quan) [inline CHANGELOG.md:7]:
```suggestion
- Update iuf-cli from 1.5.2 to 1.5.3 and cray-nlsand cray-iuf from 3.1.1 to 3.1.2 (CASMINST-6208)
```

Comment (bo-quan) [reply CHANGELOG.md:7]: missing a space after cray-nls

Comment (leliasen-hpe) [reply CHANGELOG.md]: added a space.

**Pull request merged**